exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,42 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+    // if(num < 0){
 
+    //   return true
+    // }
+    if (Math.sign(num) === -1) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+function isBetween5and10(num) {
+    if (num >= 5 && num <= 10) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+function isShortName(str) {
+    const isfullName = str.split(" ").length;
+    if (isfullName > 1) {
+        return false;
+    } else {
+        return true;
+    }
+}
+
+function startsWithD(str) {
+    if (str.startsWith("D")) {
+        return true;
+    } else {
+        return false;
+    }
+}
```

Comment:
These all work correctly. You could make your code more concise by just returning the result of your conditional statement, i.e. "return (Math.sign(num) === -1)". You don't need if/else here if you're returning true or false.